src/data/recipes.js

```diff
@@ -529,6 +551,7 @@ export const onionRings = itemify({
     [vegetableOil.id]: 1,
     [items.wheat.id]: 5,
     [soyMilk.id]: 1,
+    [salt.id]: 3,
```

Comment:
it would be neat if our recipes had optional ingredients so that we could add something like this salt, but not make it required. then if you add it you get the extra value of the salt, but if you don't have it you can still make the recipe it just has a lower value. just a thought 😄

Reply:
I like that idea! However, I think it might break the current model of all instances of an item having the same price at any given time. We could introduce item variants, but that could be unwieldy for players.

I'm open to the idea for the future. We'd just need to work through the UX and game design challenges that it might introduce. 🙂